packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -30,8 +30,8 @@ interface CompatConfig {
     dataRuntime?: string | number;
 }
 
-// N, N - 1, and N - 2
```

Comment:
This change should not be here as well. I think you have unrrelated commits in your branch that you need to remove.

Reply:
Yes, I will remove them - not sure how they got added